hyp3_gamma/dem.py

```diff
@@ -118,17 +119,9 @@ def prepare_dem_geotiff(output_name: str, geometry: ogr.Geometry, pixel_size: fl
 
                 dem_file_paths = shift_for_antimeridian(dem_file_paths, temp_path)
 
-            dem_vrt = temp_path / 'dem.vrt'
-            gdal.BuildVRT(str(dem_vrt), dem_file_paths)
-
             epsg_code = utm_from_lon_lat(centroid.GetX(), centroid.GetY())
-            gdal.Warp(
-                output_name,
-                str(dem_vrt),
-                dstSRS=f'EPSG:{epsg_code}',
-                xRes=pixel_size,
-                yRes=pixel_size,
-                targetAlignedPixels=True,
-                resampleAlg='cubic',
-                multithread=True,
-            )
+            prepare_dem_geotiff(dem_file_paths / 'dem.vrt',
+                                geometry,
+                                pixel_size=pixel_size,
+                                epsg_code=epsg_code,
+                                height_above_ellipsoid=False)
```

Comment:
With the currently proposed interface in https://github.com/ASFHyP3/hyp3-lib/pull/345 we can reduce this to:
```suggestion
            prepare_dem_geotiff(output_name,
                                geometry.Buffer(0.15),
                                pixel_size=pixel_size,
                                epsg_code=epsg_code)
```

And remove all the preceding code/helper functions/related tests other than what's needed to compute `epsg_code`